src/algorithms/ctmrg/projectors.jl

```diff
@@ -33,29 +52,65 @@ function truncation_scheme(alg::ProjectorAlgorithm, edge)
 end
 
 """
-    struct HalfInfiniteProjector{S,T}(; svd_alg=Defaults.svd_alg,
-                                      trscheme=Defaults.trscheme, verbosity=0)
+    struct HalfInfiniteProjector{S,T}
 
 Projector algorithm implementing projectors from SVDing the half-infinite CTMRG environment.
+
+## Keyword arguments
+
+* `svd_alg::Union{<:SVDAdjoint,NamedTuple}=SVDAdjoint()` : SVD algorithm including the reverse rule. See ['SVDAdjoint'](@ref).
```

Comment:
Is this the actual default? It's a bit hard to follow here so maybe it's equivalent to what's happening, but I can't see where this specific default is used.